### PR TITLE
fr-glossary/first_cpu_idle id: 227

### PR DESCRIPTION
**Pull request by ManuSquall**
suggest translation for #227 /glossary/first_cpu_idle

### CODE REVIEW

**Comment by SphinxKnight** [comment]
@ManuSquall we need to check your fork/local clone of the repository to avoid bringing 21 commits here :) 
That's not a blocker (I'll squash them before merging) but let's aim for a clean setup :)

File: files/fr/glossary/first_cpu_idle/index.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+title: Première inactivité CPU
+slug: Glossary/First_CPU_idle
+translation_of: 'Glossary/First_CPU_idle'
+---
+<p><strong>La première inactivité CPU</strong> (« <i>First CPU Idle</i> » en anglais) mesure quand une page est minimalement interactive, ou quand la fenêtre est suffisamment silencieuse pour gérer les entrées de l'utilisateur. Il s'agit d'une mesure de performance web non standard de Google. Généralement, cela se produit lorsque la plupart des éléments d'interface utilisateur (UI) visibles, mais pas nécessairement tous, sont interactifs et que l'interface utilisateur répond, en moyenne, à la plupart des entrées de l'utilisateur dans les 50 ms. Il est également connu sous le nom de <a href="/fr/docs/Glossary/First_interactive">Premier interactif</a>.</p>
```
Comment (SphinxKnight) [inline files/fr/glossary/first_cpu_idle/index.html:6]: Hmm, I'm not sure this quite conveys the performance thing in French. Personally, I'd choose something a bit more verbose like « Premier instant d'interactivité"

Comment (SphinxKnight) [reply files/fr/glossary/first_cpu_idle/index.html:6]: @ManuSquall let me know what you think of this and let's merge afterwards.

Comment (ManuSquall) [reply files/fr/glossary/first_cpu_idle/index.html:6]: Hello @SphinxKnight, I'm ok with that, guess it is more accurate in french, thank you!!

**Pull request merged**